EncodingStreamingHighAvailability/HighAvailability.JobOutputStatus/JobOutputStatusFunction.cs

```diff
@@ -0,0 +1,86 @@
+namespace HighAvailability.JobOutputStatus
+{
+    using HighAvailability.Helpers;
+    using HighAvailability.Interfaces;
+    using Microsoft.Azure.EventGrid;
+    using Microsoft.Azure.WebJobs;
+    using Microsoft.Azure.WebJobs.Extensions.EventGrid;
+    using Microsoft.Extensions.Logging;
+    using System;
+    using System.Linq;
+    using System.Threading.Tasks;
+
+    /// <summary>
+    /// Implements job output status Azure function. It is triggered by EventGrid events
```

Comment:
Can you not which event here. More explict the better

Reply:
Added more comments to each azure function, will add link to main document